docs/Guide/getting-started/creating-a-basic-app-command.mdx

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 2
+title: Creating a basic slash command
+---
+
+Creating a basic slash command is very similar to creating a basic command. The only difference is that you'll need to
+handle the command run under `chatInputRun` instead of `messageRun`.
+
+For example, let's create a `ping` command:
+
+```typescript ts2esm2cjs|{4-13}|{4-13}
+import { Command } from '@sapphire/framework';
+
+export class PingCommand extends Command {
+  public constructor(context: Command.Context, options: Command.Options) {
+    super(context, {
+      ...options,
+      name: 'ping',
+      description: 'Ping bot to see if it is alive',
+      chatInputCommand: {
+        register: true
+      }
+    });
+  }
+}
+```
+
+The above example will create a slash command (chat input command) called `ping` with the description
+`Ping bot to see if it is alive`.
+
+:::info
+
+By default sapphire registers all commands as a `Global` command which takes up to an hour to reflect in the guild
+because of discord cache limitations. It's advised to create a `Local` guild command instead during development for
+faster results. This can be done by passing `guildIds` in the command class `constructor`.
+
+```typescript ts2esm2cjs|{10}|{10}
+import { Command } from '@sapphire/framework';
+
+export class PingCommand extends Command {
+  public constructor(context: Command.Context, options: Command.Options) {
+    super(context, {
+      ...options,
+      // ...
+      chatInputCommand: {
+        register: true,
+        guildIds: ['YOUR_GUILD_ID']
+      }
+    });
+  }
+}
+```
+
+:::
+
+## Creating the `chatInputRun` method
+
+To handle the command callback when the context menu command is ran we need to override the `chatInputRun` method of
+command class.
+
+```typescript ts2esm2cjs|{8-17}|{9-17}
+import { Command } from '@sapphire/framework';
+import type { Message } from 'discord.js';
```

Comment:
```suggestion
import { isMessageInstance } from '@sapphire/discord.js-utilities';
import { Command } from '@sapphire/framework';
```

Reply:
he should also remove the information about it taking an hour to see changes globally because that's not the case anymore.

Reply:
That is still the case. Backend v2 is still WIP and not rolled out to all servers by a long long shot.

Reply:
uhm, backend v2 has 100% rollout right now